packages/bddeb

```diff
@@ -99,6 +100,27 @@ def write_debian_folder(root, templ_data, cloud_util_deps):
                              params={'build_depends': ','.join(requires)})
 
 
+def write_debian_folder_from_branch(root, templ_data, branch):
+    """Import a debian package directory from a branch."""
+    print("Importing debian/ from branch %s to %r" % (branch, root))
+    p_dumpdeb = subprocess.Popen(
+        ["git", "archive", branch, "debian"], stdout=subprocess.PIPE
```

Comment:
if your target branch does not have a directory 'debian' this will fail, do you want to either precheck or handling the expected failure ?